### DISCUSSION TITLE
Saving Files using a relative path

### DISCUSSION THREAD

**Discussion by Jedith-Freelance** [MikeMaximus/gbm#304] [Q&A] [answered]
So I've been loving using the program to get files backed up for all my games, but now I've hit a small snag that I hope can be explained.
I'm trying to save the files for Celeste, but the save files are in the same folder as the game location. The manual talks about how you can save a relative path for those save files so wherever that folder goes (moving the game between drives, etc.) it should follow it and the path will update whenever you launch the game. The problem is, I don't know how to do that. What do I put in the save path dialogue to make sure my backup is saved?
Celeste saves it's game files as <path to game>\Saves\"<save files>"

**Comment by MikeMaximus** [comment]
Hi Jedith,

Glad you like the program. 👍 

In this case you can simply put Saves in the Save Path field.  That alone isn't enough to do a backup though, you'll either need to play the game at least once so GBM can detect the Game Path, or you can set it yourself and do a manual backup right away.

Hope that helps.

**Comment by Jedith-Freelance** [comment]
Sorry, seems I didn't get the pathing right in my question.
< path-to-game >\Saves
So then my question is, if I leave the game path blank since GBM doesn't need that essentially, what does that do for the save path?
For example, if I started playing Celeste on an SSD drive, it's backing up from that drive, then I move it to my hard drive cause a storage issue and play from there, does GBM adapt to that since it can see the process is on the hard drive?
I just want to make sure when I set up my backup files stuff cause I know for a fact I'm going to not think about it after I set it up.

**Comment by MikeMaximus** [comment]
If a game changes locations, GBM will automatically update the Game Path the next time it's detected.

The only time this can be an issue is if there are multiple backup configurations with the same process name, such as multiple games using Game.exe.  In this case GBM may have trouble detecting a game if it changes locations.  This can be solved by manually clearing or setting the Game Path to the new location.

**Comment by Jedith-Freelance** [comment]
Okay, that's good to know. But I believe my original question still stands. What do I put as the save path if it's in the game's base folder? Cause if the base folder moves, the save path wouldn't match and it wouldn't save properly since the folder is no longer in that spot.

**Comment by MikeMaximus** [comment, marked as answer]
For a relative save path, GBM uses Game Path + Save Path.  So you simply put Saves as I said.
